bitnami/airflow/README.md

```diff
@@ -344,6 +344,8 @@ $ helm install my-release \
 
 The above command sets the credentials to access the Airflow web UI.
 
+> NOTE: once installed, neither username nor password can be changed using Helm CLI. If given the capability, please use the in-app features to do so.
```

Comment:
I'd recommend re-writing this as it is a bit confusing:

```
 > NOTE: Once installed for the first time, it is not possible to change username/password credentials. To change the credentials, it is necessary to remove persistent volumes (PV) and re-install.
```

@vikram-bitnami What do you think?

Reply:
WDYT of the below? I use `access credentials` because then you can use this message even for apps that use different types of credentials (tokens etc) and I mentioned both the options available for changing post-deployment

```
> NOTE: Once this chart is deployed, it is not possible to change the application's access credentials, such as usernames or passwords, using Helm. To change these application credentials after deployment, delete any persistent volumes (PVs) used by the chart and re-deploy it, or use the application's built-in administrative tools (if available).
```

Reply:
Thank you for your suggestions, I will rewrite the note.

I'm considering removing the brackets from the last sentence as it seems to fit well and it doesn't need to be a side note.
> Or use the application's built-in administrative tools if available.